wiki/People/The_Team/Beatmap_Nominators/Expectations/en.md

```diff
@@ -0,0 +1,17 @@
+# Expectations for Beatmap Nominators
+
+*See also: [Beatmap Nominator Rules](/wiki/People/The_Team/Beatmap_Nominators/Rules)*
+
+As central figures in their respective communities and osu! as a whole, [Beatmap Nominators](/wiki/People/The_Team/Beatmap_Nominators) are held to a higher standard than other users. The expectations listed here are the result of discussion within the [Nomination Assessment Team](/wiki/People/The_Team/Nomination_Assessment_Team). While it's unreasonable to perfectly align with these expectations, going far out of line can lead to warnings and/or removal from the Beatmap Nominators. All situations are judged on a case-by-case basis depending on frequency, severity, judgement, and impact.
+
+As a reminder, Beatmap Nominators are expected to do their best to adhere to the [code of conduct for modding and mapping](/wiki/Rules/Code_of_Conduct_for_Modding_and_Mapping) at all times. They should also always act within the best interest of maintaining the ranked section's quality.
+
+- **The Golden Rule.** Do not fuck up. Beatmap Nominators are expected to not act rashly and approach situations with caution. They should act with sound judgement in all circumstances.
```

Comment:
```suggestion
- **The Golden Rule: Do not fuck up.** Beatmap Nominators are expected to not act rashly and approach situations with caution. They should act with sound judgement in all circumstances.
```
so the bold part actually says something 🤔 

also I know it's meant to have the casual-but-angery tone but you could probably think of something more representative than "do not fuck up". maybe the "act with sound judgement in all circumstances" from end of this sentence?

Reply:
i would normally agree but "do not fuck up" has not had any sort of equivalent replacement that adequately encompasses all that it covers, and believe me, we've tried to figure out a different way to word it!

the unspoken onus of "not fucking up" is so powerful a thing that it's been part of the internal team ethos for like 8+ years, open to suggestions on how to reword it though

Reply:
(peppy mentioned this below in [this comment](https://github.com/ppy/osu-wiki/pull/4281#discussion_r498433487) so i guess we're addressing it)